firebase/firestore.rules

```diff
@@ -2,7 +2,7 @@ rules_version = '2';
 service cloud.firestore {
   match /databases/{database}/documents {
     match /{document=**} {
-      allow read, write: if request.auth.uid != null;
+      allow read, write: if true;
```

Comment:
Be aware that this is risky in production :)

Reply:
Okay! I'll change it before production distribution. Thank you for the information.